lib/js/components/Buttons/ToggleButton/ToggleButton.stories.mdx

```diff
@@ -0,0 +1,78 @@
+import { Meta, Story, Props } from '@storybook/addon-docs';
+
+import ToggleButton from 'design-system/lib/js/components/Buttons/ToggleButton';
```

Comment:
Shouldn't it be `./...` as in other stories?

Reply:
[`6bd2ba0` (#176)](https://github.com/bethinkpl/design-system/pull/176/commits/6bd2ba06f9921c0da22b5a2cff4582994e786564)